app/enterprise/changelog.md

```diff
@@ -19,8 +19,8 @@ nav:
   - [0.13.0 Changelog](https://github.com/Kong/kong/blob/master/CHANGELOG.md#0130---20180322)
   - [0.13.1 Changelog](https://github.com/Kong/kong/blob/master/CHANGELOG.md#0131---20180423)
 - **Kong EE 0.33** has these notices from **Kong CE 0.13**:
-  - Support for **Postgres 9.4 has been deprecated, but Kong will still start** - versions beyond 0.32 will not start with Postgres 9.4 or prior
-  - Support for **Cassandra 2.1 has been deprecated, but Kong will still start** - versions beyond 0.32 will not start with Cassandra 2.1 or prior
+  - Support for **Postgres 9.4 has been deprecated, but Kong will still start** - versions beyond 0.33 will not start with Postgres 9.4 or prior
```

Comment:
is this true? I remember @rainest saying that >=0.32 had bug since we use `CREATE TABLE IF NOT EXISTS` in our migrations, a feature of 9.5.

https://www.postgresql.org/docs/9.5/static/release-9-5.html - E.14.3.5. Object Manipulation

Reply:
@darrenjennings Are you confusing `CREATE TABLE` with `CREATE TABLE AS`? `IF NOT EXISTS` has been supported since 9.1 in `CREATE TABLE`, but only more recently in `CREATE TABLE AS`.

Reply:
You're right, it's specific to the INDEX. `CREATE INDEX IF NOT` was not supported in 9.4 but we use it.